src/reports/package/accessibilityInsightsReport.d.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+declare namespace AccessibilityInsightsReport {
+    export type Report = {
+        asHTML(): string;
+    };
+
+    export type ReportOptions = {
```

Comment:
I'm not so sure 'Options` is a proper noun for this type. It's more data you want to add to the report, certainly it's not the results from axe but contextual data to make sense of the axe result themselves. I'm sorry I don't have a better suggestion for the name.

Reply:
That's fair. I confess I didn't put much energy into this. These are just the 4 data items that we need to build the report.

Reply:
Any ideas for a better name here?

Reply:
This made me remember our `EnvironmentInfo` type.
There is also a `ScanDetailsReportModel` interface (the type is used on a `scanDetails` property somewhere else).
None of those types have exactly the properties you have here but they are kind of similar (from the usage point of view anyway).
Maybe those names/types are a good starting point?

Reply:
One point we may wish to consider is what would be a good name from the point of view of the caller. From the caller, something simple and straightforward would be preferred.

Reply:
One possibility might be to consider all five data elements a report "definition" so it could be called a ReportDefinition, or ReportRequest.

Reply:
`ReportOptions` sounds like it would select different behavior during report generation, which isn't right. Two of the fields describe the browser in which the scan for results occurred, and two fields describe the page that was scanned. Maybe something like `ScanContext` or `ResultsEnvironment`?

Reply:
I like `ScanContext`!!!